examples/experimental/sun/src/app.js

```diff
@@ -0,0 +1,114 @@
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import {StaticMap} from 'react-map-gl';
+import DeckGL from 'deck.gl';
+import {AmbientLight, DirectionalLight, LightingEffect} from '@deck.gl/core';
+import {PhongMaterial} from '@luma.gl/core';
+
+import ShadowPolygonLayer from './shadow-polygon-layer';
+import ShadowEffect from './shadow-effect/shadow-effect';
+
+// Set your mapbox token here
+const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+
+// Source data GeoJSON
+const DATA_URL =
+  'https://raw.githubusercontent.com/uber-common/deck.gl-data/master/examples/trips/buildings.json'; // eslint-disable-line
+
+const ambientLight = new AmbientLight({
+  color: [255, 255, 255],
+  intensity: 1.0
+});
+
+const dirLight = new DirectionalLight({
```

Comment:
Use `SunLight`?

Reply:
After integrating with SunLight, found the light direction is not right, need do more investigation on the sun location calculation code.